src/promptflow/tests/sdk_cli_azure_test/unittests/test_flow_entity.py

```diff
@@ -215,3 +215,30 @@ def test_resolve_requirements_for_flow(self):
 
             _, flow_dag = load_flow_dag(flow_path=flow_folder)
             assert flow_dag[ENVIRONMENT] == {"image": "python:3.8-slim"}
+
+    def test_flow_resolve_environment(self):
+        with tempfile.TemporaryDirectory() as temp:
```

Comment:
double check this test code in windows, since we often trigger the "Could not release the resource" problem in windows tests.

Reply:
Do we have a task for this?

Reply:
Offline synced, creating temp directory is not root cause of the "Could not release the resource" issue.